lighthouse-extension/gulpfile.js

```diff
@@ -155,6 +162,33 @@ gulp.task('browserify', cb => {
   runSequence('browserify-lighthouse', 'browserify-other', cb);
 });
 
+gulp.task('compilejs', () => {
+  const opts = {
+    mangle: false, // Preserve  array notation property access. Needed for artifact name lookups.
```

Comment:
Interesting how does this disrupt array notation specifically? I had thought mangle was fine unless `Function.name`/`with`/`eval` though I'm not familiar with uglify-harmony gotchas